core/src/test/java/com/homeaway/streamplatform/streamregistry/resource/BaseResourceIT.java

```diff
@@ -60,12 +60,7 @@
 import com.homeaway.digitalplatform.streamregistry.ClusterValue;
 import com.homeaway.streamplatform.streamregistry.StreamRegistryApplication;
 import com.homeaway.streamplatform.streamregistry.StreamRegistryManagedContainer;
-import com.homeaway.streamplatform.streamregistry.configuration.HealthCheckStreamConfig;
-import com.homeaway.streamplatform.streamregistry.configuration.KafkaProducerConfig;
-import com.homeaway.streamplatform.streamregistry.configuration.KafkaStreamsConfig;
-import com.homeaway.streamplatform.streamregistry.configuration.SchemaManagerConfig;
-import com.homeaway.streamplatform.streamregistry.configuration.StreamRegistryConfiguration;
-import com.homeaway.streamplatform.streamregistry.configuration.TopicsConfig;
+import com.homeaway.streamplatform.streamregistry.configuration.*;
```

Comment:
Fix IDE setting .... wildcard imports should be only in test code and static imports.